### PR TITLE
Update for 7.1 compatability

### PR DESCRIPTION
**Pull request by kbrighton**
Here's a PR that will, hopefully, help with the 7.1 port.  I've tested it on WAR and MCH, and I don't have any issues so far.  There's one part that's commented out that looks to be debugging, and I couldn't find the new signature for what was being done there, so I figure it's something that can be fixed later.

### CODE REVIEW

File: XIVComboVX/Config/ConfigWindow.cs
Comment (PrincessRTFM) [inline XIVComboVX/Config/ConfigWindow.cs:303]: None of this should be commented out; it's used in debug builds for logging what the plugin is doing, as well as useful information like the ID of the player's current job. Barring Dalamud API elements getting renamed, nothing should break in here, and it looks like you already fixed that part anyway.

File: .github/workflows/build.yml
Comment (PrincessRTFM) [inline .github/workflows/build.yml:65]: I'd rather wait until Dalamud is publicly released and _not_ change the Dalamud binary it's building against to the staging version, because staging can incorporate breaking changes and changes that are reverted before public release, and I develop against my actual game, which stays on public so that I can play.

File: XIVComboVX/XIVComboVX.csproj
Comment (PrincessRTFM) [inline XIVComboVX/XIVComboVX.csproj:6]: I use [semantic versioning](https://semver.org/), so this should only go up by one version.

**Review by PrincessRTFM — CHANGES_REQUESTED** [review]
I appreciate the help (especially finding the new sig), although I usually prefer to wait until Dalamud's public release before updating things. I develop against my actual game, and I remain on stable so that I can still play, since I also use many other plugins. Regardless, most of this looks fine (and the changes seem fairly simple aside from the sig), but there are a few things I'd need fixed before merging this in.

Also, since I _don't_ have a private beta key (and don't intend to use staging builds anyway), this will need to be tested by someone who does, and it may not be a bad idea to wait for a public build just to be sure nothing changes between the current beta and the final release, but as long as the necessary changes are made and it's tested, I'm willing to merge early.

Thank you for time and effort! I've been burnt out lately, and I know I still need to go through several jobs that just got "temporarily (I swear)" disabled because of the size of changes, so this definitely helps.

**Comment by Maxunit** [comment]
I downloaded the latest github code real quick, applied the suggested changes (the debug stuff in ConfigWindow.cs included, or it errors out) and...i only tried it on Pictomancer and Machinist real quick, but the plugin seems to load fine and seems to work fine. I will stress-test it more later today after I had some sleep.

File: XIVComboVX/XIVComboVX.csproj
Comment (kbrighton) [reply XIVComboVX/XIVComboVX.csproj:6]: Yeah, this was just so I could make new builds. :) I didn't want to bump it up fully, since it's your project.

File: XIVComboVX/Config/ConfigWindow.cs
Comment (kbrighton) [reply XIVComboVX/Config/ConfigWindow.cs:303]: Yeah, the problem here is that ClassJob doesn't contain GameData anymore, so I'll do some more research to see if there's an easier/better way of getting the abbreviation.

File: .github/workflows/build.yml
Comment (kbrighton) [reply .github/workflows/build.yml:65]: Completely fair. Once it got fully released i was going to swap this back. Thankfully, from what it looks like, it's a somewhat straightforward change.

**Comment by kbrighton** [comment]
Responded to all the comments to the code. When I have a few minutes tomorrow I'll check the docs to find the right way to fix the debug. I'm also testing some changes to xivesp, so I might have a PR for you there too. I get the burn out! It's why I was hoping that this might be helpful. :)

File: .github/workflows/build.yml
Comment (PrincessRTFM) [inline .github/workflows/build.yml:20]: Looks like you committed this by accident?

**Review by PrincessRTFM — CHANGES_REQUESTED** [review]
One line needs a fix, but then it's good to go, as long as it's been tested. If it compiles and loads and one of the jobs is confirmed functional, I'll call it good enough for now.

File: .github/workflows/build.yml
Comment (kbrighton) [reply .github/workflows/build.yml:65]: So, once dalamud gets a final release this can be swapped away from stg, but otherwise it won't build at all.  Might just set up a testing build thinger in a little bit.

**Comment by kbrighton** [comment]
Just made the change

**Review by PrincessRTFM — APPROVED** [review]
Everything looks good now!

**Pull request merged**
